src/main/java/org/opensearch/security/dlic/rest/api/FlushCacheApiAction.java

```diff
@@ -105,6 +111,8 @@ protected CType<?> getConfigType() {
 
     @Override
     protected void consumeParameters(final RestRequest request) {
-        // not needed
+        if (request.path().contains("/user/")) {
+            request.param("username");
```

Comment:
nit: is the condition required? can this be something like- https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/dlic/rest/api/CertificatesApiAction.java#L71-L75

Reply:
it should not be required as the param is only consumed if it exists, directly via request.param("username")

Reply:
the if block can be removed